lec12-datasets.Rmd

```diff
@@ -0,0 +1,189 @@
+- Rotate roles (specifically for the leader/facilitator)
+
+### Stages of group formation
+
+- "Forming-Storming-Norming-Performing-Adjourning Model"[^group_forming]
+- Essentially, groups go through stages (not always all and not always in order):
+    1. Getting to know one other
+    2. Feeling comfortable and safer, testing boundaries
+    3. Opening up, voicing opinions, potential for conflict (which can be good!)
+    4. Stronger bonds form, cooperative
+    5. Greater focus and energy on completion of project
+    6. Team ends after project is successfully completed
+- These stages can be fast-tracked by discussing norms, duties, and conduct early on
+
+[^group_forming]: See [Principles of Group Dynamics](https://open.lib.umn.edu/principlesmanagement/chapter/13-3-group-dynamics/) 
+
+### Roles and responsibilities
+
+- Leader/Facilitator's duty:
+    - Goal is to keep things running smoothly, focused on the task, and on track for time
+    - Keep everyone on topic and on task; stay aware of the time
+    - (Try to) mediate or resolve any conflicts (there will *always* be some type of conflict; how it's dealt with is what matters)
+    - (Try to) encourage everyone to participate and allow everyone a chance at talking
+- Recorder's duty:
+    - Goal is to write/type down main or important points raised or discussed when team is meeting
+    - Keep notes and files organized and orderly
+- Organizer's duty:
+    - Arrange for next meeting time and location
+    - Send reminders to members a day or two before meeting
+
+### Code of conduct
+
+- Vital to establishing boundaries and expectations of being a team member
+    - How do you want each member to treat each other?
+    - How do you deal with conflict?
+    - What is acceptable and unacceptable behaviour?
+- These are outlined in the code of conduct
+- Mostly its common sense (be kind and respectful)
+    - But its important that you as a team write out what everyone wants and agrees to
+
+## Exercise
+
+### As a group, complete these tasks
+
+<!-- show assigned groups -->
+
+- Get into your assigned groups
+- Introduce each other:
+    - Find out everyone's name and year of study
+    - Find out other things about each other (e.g. any plans for next year, etc)
+- Create a two or one word "team name". We'll use this to create a shared folder for everyone to use.
+- Assign roles to each person (these roles will be rotated in every group setting):
+    - You need: facilitator, recorder, organizer
+    - Discuss how and when roles will be rotated *(record it)*
+- Discuss and brainstorm some codes of conduct you want your team to follow *(record it)*
+- Take a few minutes, think about your own skills and what you feel are most competent in
+    - Then, share the top one or two of those skills *(record those skills)*
+- Discuss how responsibilities of each member will be decided on *(record it)*
+
+## Starting the projects
+
+### Datasets available for use
```

Comment:
What do you think of adding a [data set on bird migratory routes](https://www.pwrc.usgs.gov/BBS/?CFID=36951359&CFTOKEN=5135bf261f2f1478-471B9FA3-C648-BE26-7C2176ADADE30428) since the 60s? 

> The North American Breeding Bird Survey (BBS) web site is a cooperative effort between the U.S. Geological Survey's Patuxent Wildlife Research Center and Environment Canada's Canadian Wildlife Service to monitor the status and trends of North American bird populations. Following a rigorous protocol, BBS data are collected by thousands of dedicated participants along thousands of randomly established roadside routes throughout the continent. Professional BBS coordinators and data managers work closely with researchers and statisticians to compile and deliver these population data and population trend analyses on more than 400 bird species, for use by conservation managers, scientists, and the general public. 

They use separate tables for separate measurement, so they students will need how to join tables, which is a good exercise. There is the possibility to groupby species, habitat, routes country, state, province etc. Or we could limit it to Ontario only or similar if we thinks it get's overwhelming. [Here](https://www.pwrc.usgs.gov/BBS/State_of_the_Birds_2009.pdf) is an example reported from 2009.

Reply:
I like this option, seems like a pretty rich dataset. I didn't make my way all the way through the links to the actual data, but I trust that it is usable. And anyway if it isn't it should be obvious fairly early on.

Reply:
Ok, I'll add it to the list